aggregator_core/src/datastore/test_util.rs

```diff
@@ -130,6 +134,35 @@ impl EphemeralDatastore {
             LessSafeKey::new(UnboundKey::new(&AES_128_GCM, &self.datastore_key_bytes).unwrap());
         Crypter::new(Vec::from([datastore_key]))
     }
+
+    pub async fn downgrade(&self, target: i64) {
```

Comment:
We should stick a `#[cfg(feature = "test-util")]` on this, since we don't intend to use it at runtime.

Reply:
This module declaration in `datastore.rs` is `test-util`, so this isn't necessary.

Reply:
Ah, sorry, nevermind